### DISCUSSION TITLE
AUC - no result

### DISCUSSION THREAD

**Discussion by houpdelta** [pycaret/pycaret#3918] [Q&A]
Hello, 
I am new and I don't know if it is an error from me.
AUC doesn't work.
![Capture d’écran 2024-02-25 173622](https://github.com/pycaret/pycaret/assets/10964021/693a8fcc-21a3-4aee-b174-028dd72ff0be)
pycaret: 3.3.0
python:3.10.13

I found a main warning in log related to metric.py

Regards

**Comment by JuanBaldemarG** [comment]
This is a error.  Can you put this like a bug...in issues...Y have the same problem.
